packages/edit-site/src/components/page-pages/index.js

```diff
@@ -291,11 +318,17 @@ export default function PagePages() {
 						>
 							{ decodeEntities( item.title?.rendered ) ||
 								__( '(no title)' ) }
+							{ suffix }
 						</Link>
 					) : (
-						decodeEntities( item.title?.rendered ) ||
-							__( '(no title)' )
+						<>
```

Comment:
I think an `HStack` instead of a fragment here might allow better spacing (a small gap or could also allow to use space-between if needed)